src/Accounts/Authentication.Test/AuthenticatorsTest/DefaultSanitizerServiceTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+using Microsoft.Azure.Commands.Common.Authentication.Sanitizer.Services;
+using Microsoft.Security.Utilities;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+
+using Xunit;
+
+namespace Microsoft.Azure.Commands.Common.Authentication.Test.AuthenticatorsTest
+{
+    public class DefaultSanitizerServiceTests
+    {
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void DefaultSanitizerService_HighConfidenceSecurityModels()
```

Comment:
>DefaultSanitizerService_HighConfidenceSecurityModels [](http://example.com/codeflow?start=20&length=52)

I know that you've added a higher level test insuring this functionality is working/integrated in actual command utilization. I thought we could use these focused tests to make sure things are working at the server object level.